web/src/components/layouts/layout.tsx

```diff
@@ -346,109 +342,11 @@ export default function Layout(props: PropsWithChildren) {
         </Transition.Root>
 
         {/* Static sidebar for desktop */}
```

Comment:
a possibility is to keep this and add something in settings to always show sidebar. We should also think what the default should be, to hide or to show